src/Microsoft.OData.Edm/Schema/EdmConstants.cs

```diff
@@ -14,14 +14,26 @@ namespace Microsoft.OData.Edm
     public static class EdmConstants
     {
         /// <summary>
-        /// Version 4.0 of EDM. Corresponds to CSDL namespace "http://docs.oasis-open.org/odata/ns/edmx".
+        /// Version 4.0 of EDM, defined in CSDL namespace "http://docs.oasis-open.org/odata/ns/edmx".
         /// </summary>
         public static readonly Version EdmVersion4 = new Version(4, 0);
 
+        /// <summary>
+        /// Version 4.01 of EDM, defined in CSDL namespace "http://docs.oasis-open.org/odata/ns/edmx".
+        /// </summary>
+        public static readonly Version EdmVersion401 = new Version(4, 1);
+
         /// <summary>
         /// The current latest version of EDM.
         /// </summary>
-        public static readonly Version EdmVersionLatest = EdmVersion4;
+        public static readonly Version EdmVersionLatest = EdmVersion401;
+
+        /// <summary>
+        /// The default version of EDM to use if none is specified.
+        /// </summary>
+        public static Version EdmVersionDefault = EdmVersion4;

```

Comment:
>EdmVersion4 [](start = 50, length = 11)

Shall we make the 401 as default?

Reply:
I didn't want to make 4.01 the default because it would be a potentially breaking change for libraries that don't currently explicitly set the version.

---
In reply to: [290412054](https://github.com/OData/odata.net/pull/1476#discussion_r290412054) [](ancestors = 290412054)